rules/openssh.mk

```diff
@@ -6,6 +6,7 @@ export OPENSSH_VERSION
 
 OPENSSH_SERVER = openssh-server_$(OPENSSH_VERSION)_$(CONFIGURED_ARCH).deb
 $(OPENSSH_SERVER)_SRC_PATH = $(SRC_PATH)/openssh
+$(OPENSSH_SERVER)_DEPENDS +=  $(LIBNL3_DEV) $(LIBNL_ROUTE3_DEV)
```

Comment:
>LIBNL_ROUTE3_DEV [](http://example.com/codeflow?start=46&length=16)

Seem this fix is a valid fix, and has no relationship with the other PR 12447 .

I am worried about why this bug is not exposed before? Could you check previous build log and possible find a build bug?

Reply:
@xumia please be aware of this issue.

Reply:
> > LIBNL_ROUTE3_DEV
> 
> Seem this fix is a valid fix, and has no relationship with the other PR 12447 .
> 
> I am worried about why this bug is not exposed before? Could you check previous build log and possible find a build bug?

I checked the build log and not found build issue caused by this.
When build on local devbox or run a full build, in most case we build everything from source code, so the libnl will be build and install before openssh build.

Here is how this issue happen:
    When build on azure pipeline, we are using cache, so when change openssh code, the libnl will not build because there is no code change, also libnl will not be installed from cache. then openssh will build break because missing package.